docs/admin-sdk/docs/guide/2_api-reference/ui/mainModule.md

```diff
@@ -33,14 +33,16 @@ if (location.is(location.MAIN_HIDDEN)) {
         locationId: 'main-location-id',
     });
 
-  // If you want to provide some buttons for the smart bar of your main module
-  ui.mainModule.addSmartbarButton({
-      locationId: 'main-location-id', // locationId of your main module
-      buttonId: 'test-button', // The button id
-      label: 'Click me', // The button label
-      variant: 'primary', // The button variant
-      onClickCallback: () => {}
-  });
+    // If you want to provide some buttons for the smart bar of your main module
+    ui.mainModule.addSmartbarButton({
+        locationId: 'main-location-id', // locationId of your main module
+        buttonId: 'test-button', // The button id
+        label: 'Click me', // The button label
+        variant: 'primary', // The button variant
+        onClickCallback: () => {}
+    });
+
+    ui.mainModule.hideSmartBar({});
```

Comment:
Shouldn't it be possible to deactivate the bar using this property (`displaySearchBar`) ? https://developer.shopware.com/resources/admin-extension-sdk/api-reference/ui/mainModule.html#parameters